doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-edit-process-variables.adoc

```diff
@@ -0,0 +1,15 @@
+[id='proc-edit-process-variables_{context}']
+= Editing process variable data types
```

Comment:
Actually the feature is about correctly changing the value of the variables. Data types are not changed at all. So can we rephrase this? Thanks.